packages/augur-state/package.json

```diff
@@ -45,6 +46,9 @@
     "ethers-provider": "1.0.0-0"
   },
   "devDependencies": {
+    "@types/jest": "^24.0.3",
+    "jest": "^22.4.4",
+    "ts-jest": "^23.10.5",
```

Comment:
Are we using anything other than jest at this point? If not we can hoist these to the top level `package.json`.

Reply:
Oh makes sense. I'll do that.

Reply:
Looks like we have chai as well so I'm going to leave these here.